stable/datadog/values.yaml

```
@@ -236,6 +241,22 @@ datadog:
     #
     port: 8126
 
+    ## @param useSocketVolume - boolean - optional
+    ## Enable APM over Unix Domain Socket
+    ## ref: https://docs.datadoghq.com/agent/kubernetes/apm/
+    #
+    useSocketVolume:  # true
```

Should this not be false?
```suggestion
    useSocketVolume: false
```

---

good idea, PR updated